apps/studio/components/interfaces/QueryPerformance/QueryPerformanceFilterBar.tsx

```diff
@@ -59,73 +45,39 @@ export const QueryPerformanceFilterBar = ({
   const roles = (data ?? []).sort((a, b) => a.name.localeCompare(b.name))
 
   const onSearchQueryChange = (value: string) => {
-    setSearchInputVal(value)
-
-    if (!value || typeof value !== 'string') {
-      // if user has deleted the search query, remove it from the url
-      const { search, ...rest } = router.query
-      router.push({ ...router, query: { ...rest } })
-    } else {
-      router.push({ ...router, query: { ...router.query, search: value } })
-    }
+    setSearchParams({ search: value || '' })
   }
 
   const onFilterRolesChange = (roles: string[]) => {
     setFilters({ ...filters, roles })
     router.push({ ...router, query: { ...router.query, roles } })
   }
 
-  function getSortButtonLabel() {
-    if (sortByValue?.order === 'desc') {
-      return 'Sorted by latency - high to low'
-    } else {
-      return 'Sorted by latency - low to high'
-    }
-  }
-
-  const onSortChange = (order: 'asc' | 'desc') => {
```

Comment:
Unrelated, but I actually didn't realise that the columns were sortable by clicking on the header - wondering if we should add a tooltip or some UI indication

Reply:
I'll do a follow up updating the UI of the column headers. I think we should use the same pattern as Tables editor cols etc. I used this also in the Query Insights.